docs/jwst/tweakreg/README.rst

```diff
@@ -27,7 +27,27 @@ Alignment
 ---------
 The source catalogs for each input image are compared to each other
 and linear (affine) coordinate transformations that align these
-catalogs are derived.
+catalogs are derived.  This fit insures that all the input images
+are aligned relative to each other.  This step produces a combined
+source catalog for the entire set of input images as if they were
+combined into a single mosaic.
+
+An astrometric reference catalog then gets generated by querying
```

Comment:
I believe there's a step param that controls whether or not the alignment to GAIA is done, hence this description should be prefixed with something like "If step parameter <blee> is True, an astrometric reference catalog ..."

Reply:
Resolved in efc699a.